vowpalwabbit/core/src/reductions/lda_core.cc

```diff
@@ -32,6 +32,10 @@ VW_WARNING_STATE_POP
 #include "vw/core/vw_versions.h"
 #include "vw/io/logger.h"
 
+#if defined(__ARM_NEON)
+#include <sse2neon/sse2neon.h>
```

Comment:
This has not been made available to the `vw_core` cmake target, so the build won't be able to find this. We also don't have any CI which is going to cause this path to be exercised so we need to be careful.

The include should be added here: https://github.com/VowpalWabbit/vowpal_wabbit/blob/3ee966596575fbce66b9bef8e6d586da5b5a52af/vowpalwabbit/core/CMakeLists.txt#L372

Reply:
>  I can push a commit to the branch with this change 

That would be nice of you

Reply:
@jackgerrits FYI, I added `sse2neon` to vcpkg. Would you prefer that I add the package instead?

Reply:
Thanks for adding that, that's super helpful! There was a tiny issue with the installed path, I went ahead and opened a fix here https://github.com/microsoft/vcpkg/pull/28268. When that gets merged I will update this PR to consume.

Reply:
I'll go ahead and merge this one and we can update to consume vcpkg later.